pointmatcher/Exceptions.cpp

```diff
@@ -41,7 +41,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 template<typename T>
 PointMatcher<T>::ConvergenceError::ConvergenceError(const std::string& reason):
 	runtime_error(reason)
-{} 
+{}
+ 
+template<typename T>
+PointMatcher<T>::ConvergenceError::~ConvergenceError() throw ()
```

Comment:
This is not required and not related to the PR. I would drop it. Of course also the corresponding declaration below. (The default destructor is already virtual and equivalent to this one because ConvergenceError has a base class with a virtual destructor.)